composer.json

```diff
@@ -11,7 +11,7 @@
 		}
 	],
 	"require": {
-		"php": "^7.3 | ^7.4 | ^8.0",
+		"php": "^7.4 | ^8.0",
```

Comment:
What are your considerations for dropping support for 7.3 here? Just curious :-)

Reply:
php.net dropped support to php 7.3, it would be a wast to keep it IMO.

See more: https://www.php.net/supported-versions.php